docs/react/install.en-US.md

```diff
@@ -31,6 +31,27 @@ $ npm install antd@beta --save
 
 > **Past releases**：https://github.com/ant-design/ant-design/releases
 
+## Import in Browser
+
+We provide `antd.js` `antd.css` and `antd.min.js` `antd.min.css` under `antd/dist` in antd's npm package, in order to import all the components of antd directly. Also, you can use [npmcdn](https://npmcdn.com/).
+
+> It's not a recommended way to use built file, for you cannot get bugfixes from the dependencies of antd.
```

Comment:
Yep, this statement is not easy to understand, for it assumes that you understand the implementation of `antd`.

`antd` is based on other [npm package](https://github.com/ant-design/ant-design/blob/master/package.json#L35-L77). So, if you use `antd` as a npm module, you can get the latest version of its dependencies.